### ISSUE TITLE
Describe the process for new access requests

### ISSUE THREAD

**Issue by galargh** [ipfs/github-mgmt#35]
I think to begin with it could be as simple as a concrete set of instructions in the `ISSUE_TEMPLATE`.

Things I was wondering about:
- should we ask submitters to mention their PL managers (if applicable)
- could we extract "owners" out of the modified resources

Might be related: give org members ability to ask for reviews - https://github.com/ipfs/github-mgmt/issues/33

**Comment by BigLep** [comment]
Agreed on the value of streamlining this a bit.  Some example things I would want to capture:

1. Summary of requested permissions?
2. What is the person's affiliation with the IPFS project?
3. Justification for why want the permissions?
4. Is there anyone else that can vouch for needing these permissions (e.g., team member or manager in a PLN organization)?

I'm good to review what this template would look like.

I also think we should soon get IPDX being the CODEOWNER here.  Folks closer to the project (e.g., EngRes IPFS Stewards) should be in the loop.  We can figure that out later though, and for now I think it's good that IPDX is seeing the usecases.
